SQL_assignment/sql_assignment1

```diff
@@ -0,0 +1,82 @@
+CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+
+    Create table emp(EmpoyeeID int primary key,firstname varchar(20),lastname varchar(20),department varchar(20),salery int)
```

Comment:
Why the salary column data type is int?

Reply:
I just assumed salary to be int.
Float can also be used

Reply:
Better to user DECIMAL or NUMERIC than float.

Reply:
okey